docs/understanding/understanding.md

```diff
@@ -10,11 +10,11 @@ If you've already read our [high-level concepts](/getting_started/concepts.md) p
 
 There are a series of key steps involved in building any LLM-powered application, whether it's answering questions about your data, creating a chatbot, or an autonomous agent. Throughout our documentation, you'll notice sections are arranged roughly in the order you'll perform these steps while building your app. You'll learn about:
 
-- **[Using LLMs](/understanding/using_llms/using_llms.md)**: whether it's OpenAI or any number of hosted LLMs or a locally-run model of your own, LLMs are used at every step of the way, from indexing and storing to querying and parsing your data. LlamaIndex comes with a huge number of reliable, tested prompts and we'll also show you how to customize your own.
+- **[Using LLMs](/docs/understanding/using_llms/using_llms.md)**: whether it's OpenAI or any number of hosted LLMs or a locally-run model of your own, LLMs are used at every step of the way, from indexing and storing to querying and parsing your data. LlamaIndex comes with a huge number of reliable, tested prompts and we'll also show you how to customize your own.
```

Comment:
These actually should not have the docs prefix

Reply:
this change is repeated in a few places

Reply:
> These actually should not have the docs prefix

@logan-markewich , I see that the previous link redirected the user to the 404 page and prefix docs in the path opened the page successfully.

<img width="1411" alt="Screenshot 2023-12-20 at 8 46 35 PM" src="https://github.com/run-llama/llama_index/assets/141953346/22ad6410-51f3-4a96-9782-df1b88e8a379">

Let me know if I we can fix it in different way, I'll do that in all.